packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -87,6 +87,15 @@ function groupAccounts (accounts: SortedAccount[]): Record<GroupName, string[]>
   return ret;
 }
 
+export function getStorageMode (): {enabled: string, disabled: string} {
+  // this is a workaround to not fork https://github.com/polkadot-js/ui/tree/master/packages/ui-settings
+  // below value is the default one https://github.com/polkadot-js/ui/blob/master/packages/ui-settings/src/defaults/index.ts#L59
+  return {
+    disabled: 'on',
+    enabled: 'off'
+  };
+}
```

Comment:
I don't fully get it - why a function if the return values are static?

Reply:
This is to have a clean code point of view. Those 'on' and 'off' values are distributed in distant code areas. at least in two separate pages: settings and accounts. Imagine reading code and seeing logic: 'enable storage account if settings.get() === 'off'. this does not make sense. Therefore I grouped this hardcodes into one function which from the reader perspective makes sense, but it hides internal implementation with a comment explaining so.

Reply:
Sure, this makes sense. Just the function is not necessary - a plain object will do just fine:

```suggestion
// this is a workaround to not fork https://github.com/polkadot-js/ui/tree/master/packages/ui-settings
// below value is the default one https://github.com/polkadot-js/ui/blob/master/packages/ui-settings/src/defaults/index.ts#L59
export const StorageMode = {
    disabled: 'on',
    enabled: 'off'
  }
```
or an enum since we're using TS:
```suggestion
// this is a workaround to not fork https://github.com/polkadot-js/ui/tree/master/packages/ui-settings
// below value is the default one https://github.com/polkadot-js/ui/blob/master/packages/ui-settings/src/defaults/index.ts#L59
export enum StorageMode {
    DISABLED = 'on',
    ENABLED = 'off'
  }
```

Btw is this really right, that _disabled_ is `on` and _enabled_ is `off`? Shouldn't be the other way around?

Reply:
Yes, `off` is hardcoded in [another repo,](https://github.com/polkadot-js/ui/blob/master/packages/ui-settings/src/defaults/index.ts#L59) and I don't want to delve into forking/publishing somewhere updated fork. More forks also come with downsides.